server/src/Utopia/Web/Server.hs

```diff
@@ -63,6 +63,17 @@ redirector redirections applicationToWrap request sendResponse =
       redirectTo target = sendResponse $ responseLBS temporaryRedirect307 [("Location", target)] mempty
   in  maybe passthrough redirectTo possibleRedirection
 
+viteFudgeMiddleware :: Middleware
```

Comment:
ah, the Vite _fudge_ middleware 😂

Reply:
Ah yes, this could do with a comment actually. This was to return json imports as js, right @seanparsons ?

Reply:
Vite was returning something that ended in `.json?import` as JavaScript and with the content type for it, but our server was automagically returning it as the JSON content type because of the extension.

Reply:
Thanks. I've added a comment now reflecting this